### PR TITLE
Smtp fixups/v19

### PR DESCRIPTION
**Pull request by victorjulien**
#7576 with a formatting fix

### CODE REVIEW

**Comment by suricata-qa** [comment]
ERROR:
 

ERROR: QA failed on ips_afp_drop_chk.
  

Pipeline 8013

**Review by inashivb — APPROVED** [review]
Took me a long while to review but looks good to me. :)

File: src/util-decode-mime.c [hunk 1]
```diff
@@ -1566,7 +1566,7 @@ static int ProcessBodyLine(const uint8_t *buf, uint32_t len,
     SCLogDebug("Processing body line");
 
     /* Track length */
-    entity->body_len += len + 2; /* With CRLF */
+    entity->body_len += (len + state->current_line_delimiter_len);
```
[the remark this answers is not in the document]
Comment (victorjulien) [reply src/util-decode-mime.c:1583]: No, might be best to remove it then.

**Comment by victorjulien** [comment]
Merged in #7586

**Pull request closed without merging**